moped-editor/src/views/projects/projectView/ProjectSummary/SubprojectsTable.js

```diff
@@ -40,17 +42,44 @@ const useColumns = ({
         field: "project_id",
         editable: false,
         width: 75,
+        renderCell: ({ row }) => (
+          <Box
+            sx={{
+              display: "flex",
+              alignItems: "center",
+              height: "100%",
+              width: "100%",
+            }}
```

Comment:
I also owe an issue to tighten up styles for cells across tables. I'm looking at the project list where we use default styles in the **Full name** cell and also in the dashboard table where we have `tableCellStyles`.

That said, I'd be ok without `alignItems: center` since I think it would get closer to the other tables' styles in combination with adding the `density="comfortable"` prop on this table.